appsettings.json

```diff
@@ -13,5 +13,8 @@
       }
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "WhaleSpottingDbContextConnection": "Server=(localdb)\\mssqllocaldb;Database=whale-spotting;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
we can delete these changes (this isn't our connection string anyway)

Reply:
Actioned :)